Source/levels/trigs.cpp

```diff
@@ -754,33 +754,41 @@ bool ForcePWaterTrig()
 
 bool ForceArenaTrig()
 {
-	int *checkList = nullptr;
+	const uint16_t *checkList = nullptr;
+	size_t len;

```

Comment:
```suggestion
	size_t len = 0;
```
just to give it a default value if we did it with checkList? :P

Reply:
No need because we always assign it (or `return false`)

Reply:
Removed the default value for `checkList` as well

Reply:
that works too, just wanted consistency :p